src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx

```diff
@@ -83,27 +85,21 @@ Here are more explanations of our user management terms and how they relate to e
 
 Here are some important user management tips:
 
-* [Some basic users](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#convert-users) are able to self-serve to immediately upgrade to be [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type), which are billable users. This is possible so that if you have an incident, your team members are immediately able to work on a problem.
+* Basic users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models) are able to self-serve to immediately upgrade to be [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type), which are billable users. This is possible so that if you have an incident, your team members are immediately able to work on a problem.
```

Comment:
Instead of "This is possible so that if you have an incident, your team members are immediately able to work on a problem," which is a little awkward around cause- effect, perhaps: This enables you to immediately upgrade users in cases where you need them to work on an incident.

Also, Can you downgrade a user as well? Like, after an incident is fixed?